src/scripts/underground/UndergroundItem.ts

```diff
@@ -43,7 +43,20 @@ class UndergroundItem {
     }
 
     get imagePath() {
-        return `assets/images/underground/${this.id}.png`;
+        // Have to add extra logic here since images are all over the place in location and naming standards
+        // Maybe one day we refactor the item system to be cleaner
+        if (this.isStone()) {
+            const evostone: EvolutionStone = (ItemList[this.valueType] as EvolutionStone);
+            return evostone.imagePath;
+        } else if (this.valueType == 'Mine Egg') {
+            return `assets/images/breeding/${this.name}.png`;
+        } else {
+            return `assets/images/items/${this.name}.png`;
+        }
+    }
+
+    get underGroundImage() {
```

Comment:
Could we change this to be one word:
```suggestion
    get undergroundImage() {
```